app/views/forever_style_guide/sections/components/navigation/_nav_account_dropdown_menu.erb

```diff
@@ -44,13 +44,13 @@
           </div>
         </li>
 
-        <% if current_user.linked_ambassador.try(:active?) %>
+        <% if !current_user.linked_ambassador.try(:active?) %>
```

Comment:
I don't think you meant to commit this, it's hiding the back office link for current ambassadors